wagtail/admin/api/actions/create_alias.py

```diff
@@ -0,0 +1,52 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.create_alias import CreatePageAliasAction, CreatePageAliasIntegrityError
+from wagtail.core.models import Page
+
+from .base import APIAction
+
+
+class CreatePageAliasAPIActionSerializer(Serializer):
+    parent_page_id = fields.IntegerField(required=False)
+    recursive = fields.BooleanField(default=False, required=False)
+    update_slug = fields.CharField(required=False)
+    update_locale = fields.CharField(required=False)
```

Comment:
I think we shouldn't allow creating aliases in other locales with this API. The ``copy-for-translation`` API supports this with its ``alias`` flag.

I believe the reason this exists on the model method is because ``copy_for_translation`` will call ``create_alias`` if that flag is set.

Reply:
Could we remove this field from the API to avoid an invalid value being set?

Reply:
Sure!